### ISSUE TITLE
Will this come for classic?

### ISSUE THREAD

**Issue by Trurb** [jsb/RingMenu#11]
Not an issue.
Just want to know if you have make it work in classic when it comes :)

**Comment by jsb** [comment]
Yes, I definitely want to port this AddOn for Classic WoW.

Since I don't have Beta access, my plan is to first port this AddOn to BfA. It should then be relatively easy to adapt it for Classic WoW once it releases.

**Comment by jsb** [comment]
Due to the Secure Execution API restrictions introduced after Vanilla, I had to rewrite much of the core functionality (which I did over the last couple days). Yesterday, I was able to verify on the stress test server that the rewritten AddOn works. It's currently missing some functionality (such as animations and some styling options) but it's in a usable state.

I expect it to be ready until Classic release.
